### DISCUSSION TITLE
Decorations Request

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1536] [Art]
Do you have exciting ideas for decorations in Sunflower Land? Well now is your chance to get your ideas and designs into the game 🏆 

As we near launch of Sunflower Isles, we will begin a huge emphasis on building, exploring and decorating your land. But first, we want to ensure we have got community designed artwork available for our players ❤️ 

This is an open-ended request for in-game decorations. There are no specific requirements for what type of ideas/designs will be added.

The purpose of this Discussion is to give the artist the **freedom to design whatever items that they would want in the game.**

## Requirements

[Dimension Guide](https://docs.sunflower-land.com/contributing/for-designers/art-guidelines#dimensions)
[Shading Guide](https://docs.sunflower-land.com/contributing/for-designers/art-guidelines#shading)
[Colour Guide](https://docs.sunflower-land.com/contributing/for-designers/art-guidelines#colour-palette)
Original pieces of work. Please keep an eye out for plagiarism 💪

_By contributing to the project you agree to the terms and conditions - https://docs.sunflower-land.com/support/terms-of-service. Your item will be crafted by players and may be traded between individuals. The team and community can alter and build on top of designs contributed._

## Ideas

We are expecting exciting and innovative ideas from the artists and happy to follow your lead. Hopefully your idea can inspire boosts and utility to be added as well.

However, if you are looking for some inspiration consider designing the following:

- NPC/Mythical characters
- Farming items (tractor, scarecrows etc.)
- Rugs
- Pot plants
- Statues
- Furniture
- Magical items

## Submissions

When submitting art, please include the following:

Your farm ID (we will make sure the artists get some goodies 🏆  )
A minified version of your art piece
An example of the art used in the scene below. You should place it on the grid with the correct padding/positioning.

![example_island](https://user-images.githubusercontent.com/11745561/196328968-859dab5c-5429-47c5-a25a-02e67f2efeca.png)

**Example**

Rob the Rooster

Farm ID: 100

![rooster1](https://user-images.githubusercontent.com/11745561/196329229-01538a32-1654-41d6-a199-9feae953526f.png)
![example_island-export](https://user-images.githubusercontent.com/11745561/196329234-62dda8fb-1ee3-46d7-9d6b-b991f7bd2669.png)

## Deadline

This is an open-ended request.

We are looking to add 2-3 items per week. As always, the designer will receive an airdrop of their design and also receive a Sunflower Land Goodie pack to jump start your land expansion journey!

Good luck everyone 🥇

**Comment by LittleEins** [comment]
### News / Announcement Board(Non-NFT)
(As the LE approaches I guess we need some of this when both old and new players want to know what's new with updates or game announcement)
![news-board](https://user-images.githubusercontent.com/104184348/196466127-2f81cc7a-bbb4-438a-a544-68cd1d55c3fb.png)
![decor-request-1](https://user-images.githubusercontent.com/104184348/196466018-5d881333-6b4a-4fce-b858-41578b3a937b.png)

### Pumpkin Scarecrow
![Pumpkin-Scarecrow](https://user-images.githubusercontent.com/104184348/196502469-844f068c-a5b2-4041-8750-979c7a79d7d9.gif)
![decor-request-2](https://user-images.githubusercontent.com/104184348/196502499-d8084f58-c692-4eba-93c5-8d82f0a3cf0e.gif)

### Stone Lamp
![decors-3](https://user-images.githubusercontent.com/104184348/200525517-b65dd38d-2e74-4864-a54f-5ffeb6ff11af.gif)
![decor-3](https://user-images.githubusercontent.com/104184348/200530623-f1b3b74c-903a-4a77-b835-7efcdf340f88.gif)

#### Farm ID#73795

**Comment by digital-farms** [comment]
`Farm id: 21637`
# Skeleton-free zone
![No Skeletons](https://user-images.githubusercontent.com/105886943/198398740-85c1644e-18e5-4712-959e-03c1464ccbc0.png)
![Example_No_Skelet](https://user-images.githubusercontent.com/105886943/198399283-05dd40dc-2ada-4f96-b08d-0a5a1fd71950.png)

**Comment by NanndoSnowzera** [comment]
could someone make a war king throne for us to put on our island with skulls and tombs? I leave here the tip for decorative art that Adam is asking for LE. an iron throne on our island with war decorations would be too epic. 👀 👀 👀

**Comment by Netherzapdos** [comment]
### **Alice the Snow Leopard**
---
`Farm ID: 591`

![Alice Large Raw](https://user-images.githubusercontent.com/101855553/196377963-dfe16477-8933-46b8-8219-d8b8fc023487.gif)

![Alice Large L](https://user-images.githubusercontent.com/101855553/196378368-b6d84ffd-67a3-4d42-9ab6-4e3965273edf.gif)

One of my first few animal designs wayyy back when I first started contributing art. Couldn't find a good time at the time, but hoping this would be added to the Snow Island 👀

**Comment by ghost** [comment]
**Farm ID: 58821**

**Mailboxes** 

Preview:
![ezgif com-gif-maker (5)](https://user-images.githubusercontent.com/114235416/196450906-edd51885-70ea-48ea-95c0-03382a6e5270.gif)

**Minified:** 
_Coop inspired mailbox_
![image](https://user-images.githubusercontent.com/114235416/197409860-00f5215f-a3f4-4f9f-b3f0-7edb2ca9b9db.png)

_Barn inspired mailbox_
![image](https://user-images.githubusercontent.com/114235416/197409890-d2086237-c921-4082-8529-dffc1e98ec04.png)

_Stone mailbox_
![image](https://user-images.githubusercontent.com/114235416/197409948-6b76c715-9810-4588-8d50-f3d41776ee6b.png)

_Iron mailbox_
![image](https://user-images.githubusercontent.com/114235416/197409981-2107c0b8-1a81-4696-8c67-b9345d96a4e2.png)

_Wooden mailbox with red paint_
![image](https://user-images.githubusercontent.com/114235416/197409995-c0b0c05a-d7b4-4d6c-a9dc-c7a3b5479327.png)

_Wooden mailbox painted with pumpkin liquids_ 
![image](https://user-images.githubusercontent.com/114235416/197410003-5185574d-b9b0-4bd7-bf1c-c814cdcebcc5.png)

**Reply by adamhannigan** [reply]
This would be a great concept to receive news + airdrops.

Players would need to craft a mailbox if they wanted to receive letters + other goodies 😄

**Comment by tryjapa** [comment]
**Farm ID: 315**

Crop rugs;

**Preview:**
![crops-rugs](https://user-images.githubusercontent.com/9723813/197309076-1d2580f2-5583-4930-bd12-ec06d4caedfc.gif)


**Minified:**
Sunflower rug:
![sunflower-hug](https://user-images.githubusercontent.com/9723813/197309245-005898db-bca0-412b-95e2-218a3124431e.png)
Cauliflower rug:
![cauliflower-hug](https://user-images.githubusercontent.com/9723813/197309249-08d7f0b0-000a-4adc-8e67-31eb8f288a5a.png)
Pumpking rug:
![pumpkin-hug](https://user-images.githubusercontent.com/9723813/197309257-d71b556b-334b-4daa-bcaf-457f2cebb671.png)
Cabbage rug:
![cabagge-hug](https://user-images.githubusercontent.com/9723813/197309260-331f6e6d-35b7-40c1-8f54-1571457b23ea.png)

**Reply by adamhannigan** [reply]
Love these 😄 You may want to also post this in the new rugs proposal #1611

**Reply by tryjapa** [reply]
Thank you so much for the reminder and for the kind words, glad to hear it! I’m pretty amateur on it, but I’ve been inspired by you guys to start.

**Comment by VyroxXx** [comment]
Farm ID: 31562

Blue Demon Scarecrow:
![DemonAzulfull](https://user-images.githubusercontent.com/114549921/198156662-83f482e8-5e6e-42d7-8aa1-ca9e3a09fd18.gif)
![Espant160](https://user-images.githubusercontent.com/114549921/198156695-2aaacf11-b9ec-4d7c-8867-dc37fd258ae4.gif)
90x93
![D9093](https://user-images.githubusercontent.com/114549921/198156746-28bfbe2b-7fef-42c1-af5d-7f6ed429d42f.gif)
30x31
![D3031](https://user-images.githubusercontent.com/114549921/198156781-73967815-b840-4a86-86fa-2c0e7ffdf19f.gif)

Horse Statue:
![cavaMgrande](https://user-images.githubusercontent.com/114549921/197362717-b9bc4517-5238-4fdd-a770-46d8638adf86.png)
![cavaM160](https://user-images.githubusercontent.com/114549921/197362722-3cf4dbcb-013f-429d-ae25-f71ce81c80e9.png)
100x155
![cavaM100155](https://user-images.githubusercontent.com/114549921/197362750-50f57208-f044-47f2-b5dd-2e28713dfe6b.png)
25x39
![cavaMini2539](https://user-images.githubusercontent.com/114549921/197362765-3fa17b4b-375e-4d12-aafd-67aa310eaaa9.png)

Demon slayer golden sword:
![Espadafundo635](https://user-images.githubusercontent.com/114549921/203182837-f93823a4-d6c1-40b8-aa19-0f494711eff9.gif)
160x160
![1Esp160fundo](https://user-images.githubusercontent.com/114549921/203185203-94656647-1d91-4aaf-8a9a-ef5b68e11702.png)
43x97
![Espada4397](https://user-images.githubusercontent.com/114549921/203187141-f7df2619-6690-4155-a4f3-457ddc20461e.gif)
23x52
![Espada2352](https://user-images.githubusercontent.com/114549921/203185672-15937c12-93ef-49b3-95fb-fa16efb7bdd6.gif)

**Reply by B3atrizR3ach** [reply]
Muito legal! Bem clássico

**Reply by monge9999** [reply]
ficou massa demais! parabéns...

**Reply by MAISAbr** [reply]
ameiiii ja quero

**Reply by NanndoSnowzera** [reply]
Top dms ganhou meu UP

**Reply by VyroxXx** [reply]
> Top dms ganhou meu UP

Obrigado Nando o halloween também ficou top.

**Reply by VyroxXx** [reply]
> ameiiii ja quero

Obrigado Maisa.

**Comment by VentinRD** [comment]
Farm ID: 10350
Discord: Ventin#5150

Infamous Joel
![Infamous Joe Final](https://user-images.githubusercontent.com/103960383/198824413-b7e0ec1f-b99a-4af4-9f56-82fa14cb543c.gif)

![Infamous Joel Grid](https://user-images.githubusercontent.com/103960383/198824479-5ab06d7d-d6e2-4f63-84cd-880395c3e9d5.gif)

Lou Prince

![Lou Prince ](https://user-images.githubusercontent.com/103960383/198824692-11baaf83-7454-4c9f-b56f-600e3961963d.gif)

![Lou Prince Grid ](https://user-images.githubusercontent.com/103960383/198824696-76080f03-eabd-4279-b3a8-af2a76b0e644.gif)

Wou Prince

![Wou Prince](https://user-images.githubusercontent.com/103960383/198824705-b6764406-f688-48a7-998f-318641a7e018.gif)
![Wou Prince Grid](https://user-images.githubusercontent.com/103960383/198824709-c30cb556-8afe-44f9-85c0-e17bbc433bd4.gif)

SFL Bag

![SFL Bag](https://user-images.githubusercontent.com/103960383/198864303-4ea0f893-b51c-4615-95ff-e96f96947cf3.gif)
![sfl bag grid](https://user-images.githubusercontent.com/103960383/198864310-74f27452-cc20-414b-ac99-fc65b9e456e4.png)

**Comment by FarmerKuenster** [comment]
Meet the hungry carnivorous plant! 

So I got inspired by carnivorous plants (like venus flytraps and from the cow plant from the sims) My idea behind this plant is that it is HUNGRY and needs a food source ie. crops, chickens, other mutant plants to satisfy it. After X amount of feedings it spits out a reward for keeping it well fed! 

![hungry-plant-animation](https://user-images.githubusercontent.com/104322749/198845781-0e370e6f-d1f9-4420-a4b3-2aa5c13fe166.gif)

![landhungry-plant-on-land](https://user-images.githubusercontent.com/104322749/198845786-1b332e10-ea05-4d99-8e79-cf19e886fc47.gif)

discord: farmerkunester#1626
land ID: #32391

**Comment by VentinRD** [comment]
Farm ID: 10350
Discord: Ventin#5150

Someone asked for an iron throne, well this is my version of it.

![Gold Throne](https://user-images.githubusercontent.com/103960383/198946361-b9e45ebc-4cf1-4905-bcfa-74c61b7f75d5.png)
![Skull Pile](https://user-images.githubusercontent.com/103960383/198947565-486045dd-f29f-4e31-87c5-e9df5578b455.png)

![Gold Throne grid](https://user-images.githubusercontent.com/103960383/198982454-1b324a76-9f94-4b32-91e5-38079fa6cc59.png)

With some other decorations 
![Gold Throne and decorations Human Faction](https://user-images.githubusercontent.com/103960383/198953607-73259d54-0e83-4eb1-aa61-052797af1d90.png)
![Gold Throne and decorations Goblin Faction](https://user-images.githubusercontent.com/103960383/198953251-df5ad84c-fd87-48e6-9f4c-8d417d154633.png)

**Comment by FarmerKuenster** [comment]
Did someone ask for a tractor? Every farm needs a reliable tractor to help with all the crop harvests.

RE-DESIGNED! version 2!

![redesigned green tractor](https://user-images.githubusercontent.com/104322749/200186213-7aa0136a-e2c9-4879-b301-ebb80499d6c6.gif)
![redesigned red tractor](https://user-images.githubusercontent.com/104322749/200186228-e78636ca-542e-4d8a-aa88-2c718c01b579.gif)
![re-designed tractors-on-land](https://user-images.githubusercontent.com/104322749/200186287-ac58fb77-dcd0-46e8-b198-59b71fff3b50.gif)







discord: farmerkunester#1626
land ID: #32391

**Reply by digital-farms** [reply]
cool! but not the right perspective, look at the drawers in the back.

**Reply by FarmerKuenster** [reply]
thanks for the feedback! Your right about the perspective. I think version 2 fits the game aesthetic better now :)

**Comment by 108konstantine** [comment]
Farm ID: 151084
Discord: 108konstantine#1385
All art is in 16x20 unless mentioned.

Sheep with Border
![SheepBorder](https://user-images.githubusercontent.com/114861705/199689870-d82db80d-2dd3-4cd2-a564-da4e35d20f3f.gif)
![SheepBorderFarm](https://user-images.githubusercontent.com/114861705/199689713-374d7440-1617-44e6-9ece-a645cf82c726.gif)

Borderless Sheep
![Sheep](https://user-images.githubusercontent.com/114861705/199690390-e5d94796-52ba-44d9-9464-775ec7d1330c.gif)
![SheepFarm](https://user-images.githubusercontent.com/114861705/199689755-20b418f8-7b00-4336-87cd-593e2f2bdaf3.gif)

Evil Human King YELLOW
Shaded ![EvilKingorYellowShaded100%](https://user-images.githubusercontent.com/114861705/199799310-f19bf656-1aba-4c44-bf8e-2edecd761f57.gif)  Non-Shaded  ![EvilKingorYellow100%](https://user-images.githubusercontent.com/114861705/199799337-cdeaae44-25fe-4dbd-b50e-c5b323d22e46.gif)

![Kingor2YellowVerionsFarm](https://user-images.githubusercontent.com/114861705/199799648-86b80be1-c188-445e-9758-687371e5eb44.gif)

Evil Human King RED
Shaded ![EvilKingorRedShaded100%](https://user-images.githubusercontent.com/114861705/199799792-5baa4286-77e5-4c79-a1d5-e0ae66fa2d20.gif)  Non-Shaded ![EvilKingorRed100%](https://user-images.githubusercontent.com/114861705/199799829-4f3de478-4eae-41d5-b114-5c80774a96e9.gif)

![Kingor2RedVersionsFarm](https://user-images.githubusercontent.com/114861705/199800151-4a06946f-1226-4c85-ad02-6d17d24ac33e.gif)

GoblinKing
![GoblinKingMV100%](https://user-images.githubusercontent.com/114861705/200561535-c4c9cec9-3e85-4108-991c-00f5df029c73.gif)
![GoblinKingMV](https://user-images.githubusercontent.com/114861705/200561569-667d5847-0da8-48cb-b9d2-fa2856eca0e8.gif)

VampireBat
![Vampire100%](https://user-images.githubusercontent.com/114861705/200639323-a8285dca-9dea-4279-938e-6fa2668aab8f.gif)
![VampireFarm](https://user-images.githubusercontent.com/114861705/200639389-1ca933b6-20fd-4d7c-bb78-cb5dbd530c5e.gif)

RockstarBunnyBennington
![BunnyBennington100%](https://user-images.githubusercontent.com/114861705/201474755-4a3d2bf3-3901-4f88-af37-5834b9c0afd8.gif) 
![BunnyBennington](https://user-images.githubusercontent.com/114861705/201474782-e02272fc-94dc-4116-89de-ad5217131eeb.gif)

Water Sprinkler or Fountain. 32x32 canvas but its actually 32x17 (forgot to edit canvas)
![Sprinkler](https://user-images.githubusercontent.com/114861705/201672725-202c1139-fd9c-4bde-9541-5228bd14dca6.gif)
![Sprinkler](https://user-images.githubusercontent.com/114861705/201672756-5182372c-e9fc-4bff-bb50-4a0d774cb6e7.gif)

**Comment by Whaitte** [comment]
Hero of war
**Preview:**

![196328968-859dab5c-5429-47c5-a25a-02e67f2efeca.gif](https://user-images.githubusercontent.com/104186329/200222175-e710cfd5-c35e-46c7-a82f-82417207d770.gif)

**Minified:**
![Untitled 11-06-2022 10-01-25.gif](https://user-images.githubusercontent.com/104186329/200222268-a58809f0-f7de-4842-8036-36ac0a5ddec7.gif)

Description:_A sunflower warrior who carries with him the traces of a great war_


**Farm ID: 919**

**Comment by digital-farms** [comment]
### "Decor Fence" for the decor store
`Farm ID: 21637`

![Decor_fence_example](https://user-images.githubusercontent.com/105886943/201475434-06a6b0f6-e70a-49f9-a78f-29a14d722ab0.gif)

Horizontal: ![Decor fence - Horizontal](https://user-images.githubusercontent.com/105886943/201475292-6ee5daa2-592c-4d99-aee3-791a85423020.png)
Vertical: ![Decor fence - Vertical](https://user-images.githubusercontent.com/105886943/201475302-65251dfb-1ab4-4ea8-8e24-9326551d2e20.png)
